UI/GardenDeskForm.cs

```diff
@@ -149,6 +149,70 @@ private void SetUserRoleAccess()
             }
         }
 
+        private async void menuItemUsers_Click(object sender, EventArgs e)
+        {
+            await ShowUsersView();
+        }
+
+        #region Orest User Management
```

Comment:
Please move your region out of the "User Management Logic". That region should be deleted later.